main.py

```diff
@@ -148,14 +148,41 @@ def FillHandWithTiles(TileQueue, PlayerTiles, MaxHandSize):
 
 
 def GetScoreForWord(Word, TileDictionary):
-    Score = 0
-    for Count in range(len(Word)):
-        Score += TileDictionary[Word[Count]]
-    if len(Word) > 7:
-        Score += 20
-    elif len(Word) > 5:
-        Score += 5
-    return Score
+  done1 = 0
+  done2 = 0
```

Comment:
These should be booleans and have a name like `IsDone` or `IsChecked`

Reply:
Did it